README.md

```diff
@@ -19,22 +19,24 @@
 | [**Docs**](https://instadeepai.github.io/jumanji)
 ---
 
-<p float="left" align="center">
-  <img src="docs/env_anim/connector.gif" alt="Connector" width="30%" />
-  <img src="docs/env_anim/snake.gif" alt="Snake" width="30%" />
-  <img src="docs/env_anim/cleaner.gif" alt="Cleaner" width="30%" />
-  <img src="docs/env_anim/job_shop.gif" alt="JobShop" width="30%" />
-  <img src="docs/env_anim/bin_pack.gif" alt="BinPack" width="30%" />
-  <img src="docs/env_anim/cvrp.gif" alt="CVRP" width="30%" />
-  <img src="docs/env_anim/multi_cvrp.gif" alt="MultiCVRP" width="30%" />
-  <img src="docs/env_anim/rubiks_cube.gif" alt="RubiksCube" width="30%" />
-  <img src="docs/env_anim/graph_coloring.gif" alt="GraphColoring" width="30%" />
-  <img src="docs/env_anim/game_2048.gif" alt="Game2048" width="30%" />
-  <img src="docs/env_anim/minesweeper.gif" alt="Minesweeper" width="30%" />
-  <img src="docs/env_anim/tetris.gif" alt="Tetris" width="30%" />
-  <img src="docs/env_anim/mmst.gif" alt="MMST" width="30%" />
-  <img src="docs/env_anim/sudoku.gif" alt="Sudoku" width="30%" />
-</p>
+<div align="center">
+  <img src="docs/env_anim/connector.gif" alt="Connector" width="30%" style="margin: 5px;">
+  <img src="docs/env_anim/snake.gif" alt="Snake" width="30%" style="margin: 5px;">
+  <img src="docs/env_anim/cleaner.gif" alt="Cleaner" width="30%" style="margin: 5px;">
+</div>
+
+<div align="center">
+  <img src="docs/env_anim/job_shop.gif" alt="JobShop" width="30%" style="margin: 5px;">
+  <img src="docs/env_anim/bin_pack.gif" alt="BinPack" width="30%" style="margin: 5px;">
+  <img src="docs/env_anim/tetris.gif" alt="Tetris" width="18%" style="margin: 5px 6.5%;">
```

Comment:
Any reason for the different width/style here compared to all the others?

Reply:
To your first question, a similar comment was made above. To your second, this was to remain consistent with what has been before. We will make another update later to include all 18 GIFs in a display like on the paper. I think all environments should be on the page, otherwise it looks like there are only 9 environments implemented.

Reply:
OK but until then is there any reason not to list eg BinPack, JobShop, Tetris?